docs/ref-manual/cli/options.md

```diff
@@ -479,4 +484,23 @@ This list is incomplete.
 This option determines whether {ref}`havoc summaries <havoc-summary>` assume
 that the called method returns the correct number of return values.
 
+(-showInternalFunctions)=
+#### `--settings -showInternalFunctions`
+
+This option causes the Prover to output a list of all the potentially
+summarizable internal functions.  In some cases the Prover is unable to locate
+all internal functions, and so summaries may not be applied.  This option can
+be useful to determine whether summary is applied or not.
```

Comment:
It's maybe worth nothing that output this is applied on a per-external method basis, i.e., it's plausible (but unlikely?) that within external contract function `foo(...)` we found an call to internal function `gorp(uint)` but in another external contract function `bar(...)` we did not. Further, we do have a rough call tree relationship, if `gorp(uint)` calls the internal function `frobulate(uint)`, this is also reflected in the output.

Again, I don't know how much detail you *want* to go into here: what you have is correct, but (imho) incomplete.

Reply:
I added these details.